libqtile/backend/wayland/window.py

```diff
@@ -290,6 +295,14 @@ def _on_foreign_request_close(self, _listener, _event):
         logger.debug("Signal: foreign_toplevel_management request_close")
         self.kill()
 
+    def _on_inhibitor_destroy(self, _listener, surface: surface.Surface):
+        # We don't have reference to the inhibitor, but it doesn't really
+        # matter we only need to keep count of how many inhibitors there are
+        self._idle_inhibitors.pop()
+        _listener.remove()
```

Comment:
Could you rename `_listener` to `listener` and add a type annotation to it? The reasoning for "_listener" was because it is usually an unused variable that we want to ignore, whereas he it's being used.